images/make_drawables.py

```diff
@@ -0,0 +1,59 @@
+import sys
+import os
+import yaml
+from dpi_manager import DPIManager, ImageTypeException
+from settings import *
+from utils import (
+    ensure_dir,
+)
+
+
+def main(args):
+    test = False
+
+    if '--help' in args:
+        print("--dry-run: dry run mode, outputs to ./test folder")
```

Comment:
shouldn't this be --test, and shouldn't the script exit after printing help info?